### PR TITLE
feat(test): add --test-randomize-ordering-seed

### PR DESCRIPTION
**Pull request by felangel**
## Description

- feat(test): add --test-randomize-ordering-seed

## Type of Change

- [X] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: test/src/commands/test/test_test.dart [hunk 1]
```diff
@@ -177,6 +178,27 @@ void main() {
...
+      when<dynamic>(
+        () => argResults['test-randomize-ordering-seed'],
+      ).thenReturn('random');
+      final result = await testCommand.run();
+      expect(result, equals(ExitCode.success.code));
+      verify(
+        () => flutterTest(
+          arguments: [
+            '--test-randomize-ordering-seed',
+            'random',
```
Comment (erickzanardo) [inline test/src/commands/test/test_test.dart:191]: Should we also allow the user to pass a seed?

Comment (felangel) [reply test/src/commands/test/test_test.dart:191]: oops good catch! I had unstaged changed 😅

Comment (felangel) [reply test/src/commands/test/test_test.dart:191]: Yup they can this is just a test where we're passing random. I can add another test for a seed value.

Comment (felangel) [reply test/src/commands/test/test_test.dart:191]: Just added another test lmk what you think 👍

**Review by erickzanardo — APPROVED** [review]

**Pull request merged**